docs/conventions.md

```diff
@@ -89,6 +89,16 @@ No conventions yet
 - Contract __MUST__ exist only in case existing logic provides for an extension by the user.
 - Contract is a public API and __MUST__ follow [Spryker plugin interfaces specification](https://spryker.atlassian.net/wiki/spaces/RFC/pages/1038092073/INTEGRATED+RFC+Plugin+interface+specification).
 
+## REST API
+- Controller __SHOULD__ have no business logic.
+- Controller __SHOULD__ be named by template of task's id.
+  For task `validation:php:codestyle` - controller name is `ValidationController` and controller's action name is `validationPhpCodestyle` (task's full id in camelCase).
+- Controller __MUST__ be placed in namespace `SprykerSdk\Sdk\Presentation\RestApi\Controller\v1`. The `v1` is current version of SDK API.
```

Comment:
According to the doc provided by architect, we should use `v1.0` for versioning.  Could you discuss that with Andrea please and update the convention respectively?